src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/TypeLoaderEnvironment.DelegateMarshallerLookup.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using System.Runtime;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+using Internal.Runtime;
+using Internal.Runtime.Augments;
+using Internal.Runtime.CompilerServices;
+
+using Internal.NativeFormat;
+using Internal.TypeSystem;
+
+namespace Internal.Runtime.TypeLoader
+{
+    public sealed partial class TypeLoaderEnvironment
+    {
+        public unsafe IntPtr TryGetMarshallerForDelegate(RuntimeTypeHandle delegateTypeHandle)
```

Comment:
I would personally have a preference to keep this in System.Private.Interop, but I have no objections to having this in System.Private.TypeLoader